bin/dfx-sns-demo

```diff
@@ -84,9 +84,12 @@ sleep 1
 
 ./bin/dfx-sns-demo-mksns --network "$DFX_NETWORK"
 sleep 1
-dfx-sns-sale-buy --network "$DFX_NETWORK"
+./bin/dfx-sns-sale-buy --network "$DFX_NETWORK"
```

Comment:
Note: On line 5 the bin dir is placed at the front of the path, so all these `./bin` prefixes can actually be deleted.  However it doesn't really matter either way and your change makes the commands consistent.  Nimzowitsch liked to over-protect key squares, why not over-protect running the right program?

Reply:
It's not relevant for this PR but at some stage I would drop all the `./bin`.  This is not a blocker and not a request for change.